### ISSUE TITLE
[YouTube] Unable to extract video title as of 2019-07-29 23:36 GMT

### ISSUE THREAD

**Issue by fmillion** [ytdl-org/youtube-dl#21935]
## Checklist

- [x] I'm reporting a broken site support issue
- [x] I've verified that I'm running youtube-dl version **2019.07.27**
- [x] I've checked that all provided URLs are alive and playable in a browser
- [x] I've checked that all URLs and arguments with special characters are properly quoted or escaped
- [x] I've searched the bugtracker for similar bug reports including closed ones
- [x] I've read bugs section in FAQ

## Verbose log

```
root@odroidxu4:/mnt# youtube-dl -v "https://www.youtube.com/watch?v=QLJSdNYcdpk"
[debug] System config: []
[debug] User config: ['--external-downloader', 'aria2c', '--external-downloader-args', '-c -j 3 -x 3 -s 3 -k 1M --file-allocation=none', '-o', '%(upload_date)s %(title)s [%(id)s].%(ext)s', '--sub-langs', 'en', '--embed-subs', '--embed-thumbnail', '--add-metadata', '--xattrs', '--write-sub', '--write-auto-sub', '--write-info-js']
[debug] Custom config: []
[debug] Command-line args: ['-v', 'https://www.youtube.com/watch?v=QLJSdNYcdpk']
[debug] Encodings: locale UTF-8, fs utf-8, out UTF-8, pref UTF-8
[debug] youtube-dl version 2019.07.27
[debug] Python version 3.6.8 (CPython) - Linux-4.14.133-odroidxu4-armv7l-with-Ubuntu-18.04-bionic
[debug] exe versions: ffmpeg 3.4.6, ffprobe 3.4.6
[debug] Proxy map: {}
[youtube] QLJSdNYcdpk: Downloading webpage
[youtube] QLJSdNYcdpk: Downloading video info webpage
WARNING: Unable to extract video title
[youtube] QLJSdNYcdpk: Looking for automatic captions
[debug] Default format spec: bestvideo+bestaudio/best
[info] Writing video subtitles to: 20150801 _ [QLJSdNYcdpk].en.vtt
[info] Writing video description metadata as JSON to: 20150801 _ [QLJSdNYcdpk].info.json
[youtube] QLJSdNYcdpk: Downloading thumbnail ...
[youtube] QLJSdNYcdpk: Writing thumbnail to: 20150801 _ [QLJSdNYcdpk].jpg
WARNING: Requested formats are incompatible for merge and will be merged into mkv.
[debug] Invoking downloader on 'https://r12---sn-bvvbax-hn2d.googlevideo.com/videoplayback?expire=1564464269&ei=LYA_XdvKLKmWir4PzbmGUA&ip=24.159.198.206&id=o-ABfuh4hWnF4kx8T691ZTv33R6FLbk2C_Gi4MX7nkFj58&itag=136&aitags=133%2C134%2C135%2C136%2C160%2C242%2C243%2C244%2C247%2C278&source=youtube&requiressl=yes&mm=31%2C26&mn=sn-bvvbax-hn2d%2Csn-qxoedn7z&ms=au%2Conr&mv=m&mvi=11&nh=EAQ%2C&pl=20&initcwndbps=2163750&mime=video%2Fmp4&gir=yes&clen=47015096&dur=623.399&lmt=1540851766270616&mt=1564442561&fvip=3&keepalive=yes&c=WEB&txp=5533432&sparams=expire%2Cei%2Cip%2Cid%2Caitags%2Csource%2Crequiressl%2Cmime%2Cgir%2Cclen%2Cdur%2Clmt&sig=ALgxI2wwRgIhAJmAB7T1vIOSLwRFqV_rwKkmrJRo0OUMUqbmPWZ90mLkAiEAq0P0yDtE1pPLKp018ZT0oBquXrs7-CD5__x_6SiDclY%3D&lsparams=mm%2Cmn%2Cms%2Cmv%2Cmvi%2Cnh%2Cpl%2Cinitcwndbps&lsig=AHylml4wRQIgLTaSa1ZIo2iQ-dsSrbPSaKHCbLNateEdgOKoByko8cwCIQDsNZOISt2sh1-m_5mbZtrdbf3rjQTW6AhTmDUHxt6M-g%3D%3D&ratebypass=yes'
[download] Destination: 20150801 _ [QLJSdNYcdpk].f136.mp4
[debug] aria2c command line: aria2c -c -c -j 3 -x 3 -s 3 -k 1M --file-allocation=none --out '20150801 _ [QLJSdNYcdpk].f136.mp4.part' --header 'User-Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/73.0.3674.1 Safari/537.36' --header 'Accept-Charset: ISO-8859-1,utf-8;q=0.7,*;q=0.7' --header 'Accept: text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8' --header 'Accept-Encoding: gzip, deflate' --header 'Accept-Language: en-us,en;q=0.5' --check-certificate=true -- 'https://r12---sn-bvvbax-hn2d.googlevideo.com/videoplayback?expire=1564464269&ei=LYA_XdvKLKmWir4PzbmGUA&ip=24.159.198.206&id=o-ABfuh4hWnF4kx8T691ZTv33R6FLbk2C_Gi4MX7nkFj58&itag=136&aitags=133%2C134%2C135%2C136%2C160%2C242%2C243%2C244%2C247%2C278&source=youtube&requiressl=yes&mm=31%2C26&mn=sn-bvvbax-hn2d%2Csn-qxoedn7z&ms=au%2Conr&mv=m&mvi=11&nh=EAQ%2C&pl=20&initcwndbps=2163750&mime=video%2Fmp4&gir=yes&clen=47015096&dur=623.399&lmt=1540851766270616&mt=1564442561&fvip=3&keepalive=yes&c=WEB&txp=5533432&sparams=expire%2Cei%2Cip%2Cid%2Caitags%2Csource%2Crequiressl%2Cmime%2Cgir%2Cclen%2Cdur%2Clmt&sig=ALgxI2wwRgIhAJmAB7T1vIOSLwRFqV_rwKkmrJRo0OUMUqbmPWZ90mLkAiEAq0P0yDtE1pPLKp018ZT0oBquXrs7-CD5__x_6SiDclY%3D&lsparams=mm%2Cmn%2Cms%2Cmv%2Cmvi%2Cnh%2Cpl%2Cinitcwndbps&lsig=AHylml4wRQIgLTaSa1ZIo2iQ-dsSrbPSaKHCbLNateEdgOKoByko8cwCIQDsNZOISt2sh1-m_5mbZtrdbf3rjQTW6AhTmDUHxt6M-g%3D%3D&ratebypass=yes'

07/29 23:24:32 [NOTICE] Downloading 1 item(s)
[#a701dd 15MiB/44MiB(34%) CN:3 DL:9.1MiB ETA:3s]
07/29 23:24:34 [NOTICE] CUID#9 - Redirecting to https://r3---sn-vgqsenls.googlevideo.com/videoplayback?expire=1564464269&ei=LYA_XdvKLKmWir4PzbmGUA&ip=24.159.198.206&id=o-ABfuh4hWnF4kx8T691ZTv33R6FLbk2C_Gi4MX7nkFj58&itag=136&aitags=133%2C134%2C135%2C136%2C160%2C242%2C243%2C244%2C247%2C278&source=youtube&requiressl=yes&mime=video%2Fmp4&gir=yes&clen=47015096&dur=623.399&lmt=1540851766270616&fvip=3&keepalive=yes&c=WEB&txp=5533432&sparams=expire%2Cei%2Cip%2Cid%2Caitags%2Csource%2Crequiressl%2Cmime%2Cgir%2Cclen%2Cdur%2Clmt&sig=ALgxI2wwRgIhAJmAB7T1vIOSLwRFqV_rwKkmrJRo0OUMUqbmPWZ90mLkAiEAq0P0yDtE1pPLKp018ZT0oBquXrs7-CD5__x_6SiDclY%3D&ratebypass=yes&redirect_counter=1&rm=sn-bvvbax-hn2del&req_id=3ed470192308cf9&cms_redirect=yes&ipbypass=yes&mm=30&mn=sn-vgqsenls&ms=nxu&mt=1564442551&mv=m&mvi=2&pl=20&lsparams=ipbypass,mm,mn,ms,mv,mvi,pl&lsig=AHylml4wRQIgEYFbKkhAreqj_uochzLlo9wCw0qYLEqZm7-uGj6H0HICIQCgoHeaqrnusFAvvQzNl3J68WEnOSCosashA0xJivk2RA==
[#a701dd 44MiB/44MiB(99%) CN:2 DL:6.6MiB]
07/29 23:24:39 [NOTICE] Download complete: /mnt/20150801 _ [QLJSdNYcdpk].f136.mp4.part

Download Results:
gid   |stat|avg speed  |path/URI
======+====+===========+=======================================================
a701dd|OK  |   6.2MiB/s|/mnt/20150801 _ [QLJSdNYcdpk].f136.mp4.part

Status Legend:
(OK):download completed.
[aria2c] Downloaded 47015096 bytes
[download] 100% of 44.84MiB in 00:07
[debug] Invoking downloader on 'https://r12---sn-bvvbax-hn2d.googlevideo.com/videoplayback?expire=1564464269&ei=LYA_XdvKLKmWir4PzbmGUA&ip=24.159.198.206&id=o-ABfuh4hWnF4kx8T691ZTv33R6FLbk2C_Gi4MX7nkFj58&itag=251&source=youtube&requiressl=yes&mm=31%2C26&mn=sn-bvvbax-hn2d%2Csn-qxoedn7z&ms=au%2Conr&mv=m&mvi=11&nh=EAQ%2C&pl=20&initcwndbps=2163750&mime=audio%2Fwebm&gir=yes&clen=9398683&dur=623.421&lmt=1540855614453344&mt=1564442561&fvip=3&keepalive=yes&c=WEB&txp=5511222&sparams=expire%2Cei%2Cip%2Cid%2Citag%2Csource%2Crequiressl%2Cmime%2Cgir%2Cclen%2Cdur%2Clmt&sig=ALgxI2wwRQIgCOZ-IyNwitmVCNt7GHrNr_ps7moZLL9pslwVtPl9OmsCIQCer0NM9do3qVWr_UZcPsDo8Xd4-YVKsncG4xGOlp5OQg%3D%3D&lsparams=mm%2Cmn%2Cms%2Cmv%2Cmvi%2Cnh%2Cpl%2Cinitcwndbps&lsig=AHylml4wRQIgLTaSa1ZIo2iQ-dsSrbPSaKHCbLNateEdgOKoByko8cwCIQDsNZOISt2sh1-m_5mbZtrdbf3rjQTW6AhTmDUHxt6M-g%3D%3D&ratebypass=yes'
[download] Destination: 20150801 _ [QLJSdNYcdpk].f251.webm
[debug] aria2c command line: aria2c -c -c -j 3 -x 3 -s 3 -k 1M --file-allocation=none --out '20150801 _ [QLJSdNYcdpk].f251.webm.part' --header 'User-Agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/73.0.3674.1 Safari/537.36' --header 'Accept-Charset: ISO-8859-1,utf-8;q=0.7,*;q=0.7' --header 'Accept: text/html,application/xhtml+xml,application/xml;q=0.9,*/*;q=0.8' --header 'Accept-Encoding: gzip, deflate' --header 'Accept-Language: en-us,en;q=0.5' --check-certificate=true -- 'https://r12---sn-bvvbax-hn2d.googlevideo.com/videoplayback?expire=1564464269&ei=LYA_XdvKLKmWir4PzbmGUA&ip=24.159.198.206&id=o-ABfuh4hWnF4kx8T691ZTv33R6FLbk2C_Gi4MX7nkFj58&itag=251&source=youtube&requiressl=yes&mm=31%2C26&mn=sn-bvvbax-hn2d%2Csn-qxoedn7z&ms=au%2Conr&mv=m&mvi=11&nh=EAQ%2C&pl=20&initcwndbps=2163750&mime=audio%2Fwebm&gir=yes&clen=9398683&dur=623.421&lmt=1540855614453344&mt=1564442561&fvip=3&keepalive=yes&c=WEB&txp=5511222&sparams=expire%2Cei%2Cip%2Cid%2Citag%2Csource%2Crequiressl%2Cmime%2Cgir%2Cclen%2Cdur%2Clmt&sig=ALgxI2wwRQIgCOZ-IyNwitmVCNt7GHrNr_ps7moZLL9pslwVtPl9OmsCIQCer0NM9do3qVWr_UZcPsDo8Xd4-YVKsncG4xGOlp5OQg%3D%3D&lsparams=mm%2Cmn%2Cms%2Cmv%2Cmvi%2Cnh%2Cpl%2Cinitcwndbps&lsig=AHylml4wRQIgLTaSa1ZIo2iQ-dsSrbPSaKHCbLNateEdgOKoByko8cwCIQDsNZOISt2sh1-m_5mbZtrdbf3rjQTW6AhTmDUHxt6M-g%3D%3D&ratebypass=yes'

07/29 23:24:39 [NOTICE] Downloading 1 item(s)
[#4af490 8.6MiB/8.9MiB(96%) CN:1 DL:131KiB ETA:2s]
07/29 23:24:51 [NOTICE] Download complete: /mnt/20150801 _ [QLJSdNYcdpk].f251.webm.part

Download Results:
gid   |stat|avg speed  |path/URI
======+====+===========+=======================================================
4af490|OK  |   846KiB/s|/mnt/20150801 _ [QLJSdNYcdpk].f251.webm.part

Status Legend:
(OK):download completed.
[aria2c] Downloaded 9398683 bytes
[download] 100% of 8.96MiB in 00:11
[ffmpeg] Merging formats into "20150801 _ [QLJSdNYcdpk].mkv"
[debug] ffmpeg command line: ffmpeg -y -loglevel repeat+info -i 'file:20150801 _ [QLJSdNYcdpk].f136.mp4' -i 'file:20150801 _ [QLJSdNYcdpk].f251.webm' -c copy -map 0:v:0 -map 1:a:0 'file:20150801 _ [QLJSdNYcdpk].temp.mkv'
Deleting original file 20150801 _ [QLJSdNYcdpk].f136.mp4 (pass -k to keep)
Deleting original file 20150801 _ [QLJSdNYcdpk].f251.webm (pass -k to keep)
[ffmpeg] Adding metadata to '20150801 _ [QLJSdNYcdpk].mkv'
[debug] ffmpeg command line: ffmpeg -y -loglevel repeat+info -i 'file:20150801 _ [QLJSdNYcdpk].mkv' -c copy -metadata title=_ -metadata date=20150801 -metadata 'description=Visit 8-bit Keys here:
https://www.youtube.com/c/8BitKeys/videos

Visit me on Facebook:
https://www.facebook.com/the8bitguy

Synths that use the YM3812

Yamaha PSR-11     49-keys   16-sounds  (1986)
Yamaha PSR-12     49-keys   32-sounds  (1987)
Yamaha PSR-31*   61-keys   16-sounds  (1991)
Yamaha PSR-32*   61-keys   32-sounds  (1987)
Yamaha PSS-460   49-keys   21-sounds  (1986)
Yamaha PSS-470   49-keys    21-sounds  (1987)
Yamaha PSS-560* 49-keys    21-sounds  (1986)
Yamaha PSS-570* 49-keys   21-sounds  (1987)

*While it does use the YM3812, it also uses an additional chip for drums.

Synths that use the YM2413 (cost reduced YM3812)

Yamaha PSR-6  49-keys  100-sounds  (1994)
Yamaha PSS-140  37-keys  100-sounds  (1988)
Yamaha PSS-170  44-keys  100-sounds  (1986)
Yamaha PSS-270  49-keys  100-sounds  (1986)' -metadata 'comment=Visit 8-bit Keys here:
https://www.youtube.com/c/8BitKeys/videos

Visit me on Facebook:
https://www.facebook.com/the8bitguy

Synths that use the YM3812

Yamaha PSR-11     49-keys   16-sounds  (1986)
Yamaha PSR-12     49-keys   32-sounds  (1987)
Yamaha PSR-31*   61-keys   16-sounds  (1991)
Yamaha PSR-32*   61-keys   32-sounds  (1987)
Yamaha PSS-460   49-keys   21-sounds  (1986)
Yamaha PSS-470   49-keys    21-sounds  (1987)
Yamaha PSS-560* 49-keys    21-sounds  (1986)
Yamaha PSS-570* 49-keys   21-sounds  (1987)

*While it does use the YM3812, it also uses an additional chip for drums.

Synths that use the YM2413 (cost reduced YM3812)

Yamaha PSR-6  49-keys  100-sounds  (1994)
Yamaha PSS-140  37-keys  100-sounds  (1988)
Yamaha PSS-170  44-keys  100-sounds  (1986)
Yamaha PSS-270  49-keys  100-sounds  (1986)' -metadata 'purl=https://www.youtube.com/watch?v=QLJSdNYcdpk' -metadata 'artist=The 8-Bit Guy' 'file:20150801 _ [QLJSdNYcdpk].temp.mkv'
[ffmpeg] Embedding subtitles in '20150801 _ [QLJSdNYcdpk].mkv'
[debug] ffmpeg command line: ffmpeg -y -loglevel repeat+info -i 'file:20150801 _ [QLJSdNYcdpk].mkv' -i 'file:20150801 _ [QLJSdNYcdpk].en.vtt' -map 0 -c copy -map -0:s -map -0:d -map 1:0 -metadata:s:s:0 language=eng 'file:20150801 _ [QLJSdNYcdpk].temp.mkv'
Deleting original file 20150801 _ [QLJSdNYcdpk].en.vtt (pass -k to keep)
ERROR: Only mp3 and m4a/mp4 are supported for thumbnail embedding for now.
Traceback (most recent call last):
  File "/usr/local/lib/python3.6/dist-packages/youtube_dl/YoutubeDL.py", line 2054, in post_process
    files_to_delete, info = pp.run(info)
  File "/usr/local/lib/python3.6/dist-packages/youtube_dl/postprocessor/embedthumbnail.py", line 91, in run
    raise EmbedThumbnailPPError('Only mp3 and m4a/mp4 are supported for thumbnail embedding for now.')
youtube_dl.postprocessor.embedthumbnail.EmbedThumbnailPPError: Only mp3 and m4a/mp4 are supported for thumbnail embedding for now.
```

## Description

As of 23:36 GMT on 2019-07-29, youtube-dl is unable to extract video titles on YouTube. Videos download with the title "_". 

This is a new issue. As of yesterday, youtube-dl was working properly.

(Note: In my log, please ignore errors with embedding/thumbnails/etc. The video does download successfully, just without the title.)

**Comment by shirishag75** [comment]
@fmillion, why did you close the bug. I saw the same behavior at my end just sometime back as well. Seems youtube.com has done some changes.

**Comment by wlerin** [comment]
It's been fixed #8dbf751aa241475dd8a7a6d3040713b5874fd057 but the fix isn't on pypi yet.
